prboom2/src/dsda/console.c

```diff
@@ -26,6 +26,8 @@
 
 #include "console.h"
 
+#include "m_cheat.h"
```

Comment:
Could you put this above `#include "m_menu.h"`? I try to organize the header includes in order by group: doom / prboom+ files first, then dsda files, then the current file.